core/storage/classifier/gae_models.py

```diff
@@ -95,19 +100,19 @@ def create(
                 data used by the classifier.
 
         Returns:
-            ID of the new ClassifierModel entry.
+            ID of the new ClassifierDataModel entry.
 
         Raises:
             Exception: A model with the same ID already exists.
         """
 
         instance_id = cls._generate_id(exp_id)
```

Comment:
`job_request_id` is generated with `<exp_id>.random_hash` (see [this](https://docs.google.com/document/d/1kACmivEvZVlTYBdqxy-mmdTi3sg_NbkZdS10T-ya-pg/edit#heading=h.46ch36jfh9do), Anmol has mentioned it there). So we dont have to worry about how `classifier_id` is generated.